index.js

```diff
@@ -26,6 +27,10 @@ const plugin = fp(
     if (typeof opts.authContext !== 'undefined') {
       app.graphql.addHook('preExecution', auth.authContextHook.bind(auth))
     }
+
+    if (opts.namespace && opts.authDirective) {
+      filterSchema(app, authSchema, opts)
```

Comment:
Have you accounted for gateway mode? What happens when the gateway schema is refreshed?

Reply:
Fixed!